src/app/app.component.ts

```diff
@@ -36,6 +36,10 @@ export class AppComponent implements OnInit {
     this.updateSidenav();
   }
 
+  get mode() {
+    return !this.isMobileDisplay ? 'side' : 'over';
+  }
```

Comment:
this could be a static property instead of a getter? Where `isMobileDisplay` changes?

Angular treats a getter same way as a function, it runs constantly

If not... I suggest invert ternary to remove negation
```suggestion
  get mode() {
    return this.isMobileDisplay ? 'over' : 'side';
  }
```

Reply:
Thanks for pointing out the negated ternary. I was trying out some variations and was so burned out last night that I forgot to change it back before pushing.

Regarding the getter, I've seen a couple of examples that did it like that, and I thought it was a correct way of doing it. I wrote it like this for simplicity, to avoid having to call an updateMode function manually, but if the getter is being constantly run, then you're completely right and it should be a property instead.